content/millennium/r4/scheduling/slot.md

```diff
@@ -193,6 +193,68 @@ List an individual Slot by its id:
 
 The common [errors] and [OperationOutcomes] may be returned.
 
+## Patch
+
+<%= beta_tag(action: true) %>
+
+Known Issues:
+
+* A 500 Internal Server Error is thrown when an invalid version id is used in the If-Match Header
```

Comment:
I was under the impression we wanted this to be within the beta tag?

Reply:
added [here](https://github.com/cerner/fhir.cerner.com/pull/608/commits/293de4a74bdac6b27c4ef8f6059e60d6d4255df8)